### PR TITLE
Add telco page

### LINKED ISSUE #2390: telco page build

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add Telco page 

**Contact us link needed for 'hero' not to be merged until this is added**

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/telecommunications](http://0.0.0.0:8001/telecommunications)
- Make sure the page is as per [copy doc](https://docs.google.com/document/d/15FY_IaT39V81FQcoSoEJpmRRjBX80PJ7uxjgzHnDDws/edit)

## Issue / Card

Fixes #2390 
Doc: https://docs.google.com/document/d/15FY_IaT39V81FQcoSoEJpmRRjBX80PJ7uxjgzHnDDws/edit

### CODE REVIEW

**Comment by grahambancroft** [comment]
@frankiegranato I wasn't entirely sure what to do with the "The complete cloud automation toolset" type sections, I tried them as cards also, but it felt like there was too much content, it'd be great if either you or @anthonydillon could advise.

File: templates/telecommunications/index.html [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      <p>Our team of specialists operates production CSP environments around the globe, covering the entire environment life-cycle, from deployment to upgrades and onboarding virtual network functions &mdash; enabling you to realise the true potential of your cloud.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>Why choose Canonical?</h2>
+    <ul class="p-list is-split">
+      <li class="p-list__item is-ticked">Keep CAPEX and OPEX costs down with virtualisation and automation</li>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: Could you add abbreviations here please?

File: templates/telecommunications/index.html [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+      <li class="p-list__item is-ticked">Peace of mind with a pre-tested solution</li>
+      <li class="p-list__item is-ticked">Get started quicker with our speedy onboarding process</li>
+      <li class="p-list__item is-ticked">High-availability SLAs aligned with accepted industry practices</li>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: Please add abbreviations here please.

File: templates/telecommunications/index.html [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+      <li class="p-list__item is-ticked">Stay ahead with easy upgrades between Openstack versions and fast access to the latest Openstack features with 100% upstream integration</li>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: It's OpenStack, with capital S

File: templates/telecommunications/index.html [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>The complete cloud automation toolset</h2>
+      <p>Operational discipline &mdash; built on automation and repeatability &mdash; has long been crucial to the profitability of CSPs. Canonical applies the same principles to the way clouds are built and run.  From design to deployment and operations, every aspect is managed with Canonical&rsquo;s proven automation toolset, including provisioning tool MAAS, management tool Landscape, and application modeling tool Juju.</p>
+      <ul class="p-list">
+        <li><a href="/cloud/juju">Find out more about Juju&nbsp;&rsaquo;</a></li>
+        <li><a href="/server/maas">Find out more about MaaS&nbsp;&rsaquo;</a></li>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: It's all caps: MAAS

File: templates/telecommunications/index.html [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>NFVI cloud build</h2>
+      <p>Canonical&rsquo;s Foundation Cloud Build for Telcos is a pre-tested, pre-integrated Openstack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: OpenStack

File: templates/telecommunications/index.html [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about NFVI cloud build</span>&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>A telco-grade managed cloud</h2>
+      <p>Canonical&rsquo;s Foundation Cloud Build for Telcos is a pre-tested, pre-integrated Openstack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: OpenStack

File: templates/telecommunications/index.html [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about Canonical&rsquo;s Foundation Cloud Build</span>&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Smart edge: Internet of Things, switches and CPEs</h2>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: Abbreviations please

File: templates/telecommunications/index.html [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-12">
+        <h2 class="p-muted-heading u-align--center">A selection of our telecommunications clients</h2>
+        <ul class="p-inline-images">
+          <li class="p-inline-images__item">
+            <img src="{{ ASSET_SERVER_URL }}25de1877-Tele2.svg" alt="Tele2" />
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}03a06060-logo-deutschetelekom.svg" width="242" alt="Deutshe Telekom logo">
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: Please remove logo from the alt text

File: templates/telecommunications/index.html [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}b6e008be-Colt-Logo.svg" alt="Colt logo" />
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}481063cd-pccw.svg" alt="PCCW logo" />
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}071c255b-AT%26T_logo.svg" alt="AT&amp;T logo" />
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: You don't need to encode ampersands in the alt text

**Comment by anthonydillon** [comment]
Personally I think the "The complete cloud automation toolset" section looks ok but will need images on the right.

**Comment by spencerbygraves** [comment]
I agree the layout of the automation section is fine. @frankiegranato do you know of any assets other than pictograms?

File: templates/telecommunications/index.html (hunk 1 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done

File: templates/telecommunications/index.html (hunk 2 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done

File: templates/telecommunications/index.html (hunk 4 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done and updated in copy doc

File: templates/telecommunications/index.html (hunk 6 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done and updated in copy doc

File: templates/telecommunications/index.html (hunk 5 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done and updated in copy doc

File: templates/telecommunications/index.html (hunk 7 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done

File: templates/telecommunications/index.html (hunk 8 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Removed form all

File: templates/telecommunications/index.html (hunk 9 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done

**Comment by grahambancroft** [comment]
@anthonydillon, @spencerbygraves I meant that type of section, so including NFVI cloud build, A telco-grade managed cloud and Smart edge: Internet of Things, switches and CPEs, was thinking we could do more than just pictos but wasn't sure if there was any other options

File: templates/telecommunications/index.html [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+
+{% block title %}Ubuntu solutions for telecommunications{% endblock %}
+
+{% block content %}
+<section class="p-strip--accent is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>Ubuntu for telcos</h1>
+      <p>As an established leader in cloud computing, Canonical let&rsquo;s you apply the webscale savings and best practice  that are necessary to succeed in the telco world. By using established open source software to automate the deployment and operation of NFVI and IoT solutions, we can help you become even more agile while reducing your costs and time to market.</p>
+      <p><a class="p-button--positive p-link--external" href="https://www.brighttalk.com/webcast/6793/290067" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Watch Cloud to Edge webinar', 'eventLabel' : 'Watch Cloud to Edge webinar', 'eventValue' : undefined });" >Watch Cloud to Edge webinar</a></p>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: Shouldn't there be a `Contact us` second CTA here.

File: templates/telecommunications/index.html [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about NFVI cloud build</span>&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>A telco-grade managed cloud</h2>
+      <p>Canonical&rsquo;s Foundation Cloud Build for Telcos is a pre-tested, pre-integrated OpenStack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: This should be:
> A number of telcos rely on Canonical’s fully managed cloud service. Get Telco Grade SLAs and predictable operational costs for your OpenStack cloud, on your servers at your locations. We build, operate, scale and upgrade your cloud, while you concentrate on building your applications in flexible and agile environments.

File: templates/telecommunications/index.html [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about Canonical&rsquo;s Foundation Cloud Build</span>&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: `/cloud/managed-cloud`

File: templates/telecommunications/index.html [hunk 13]
```diff
@@ -0,0 +1,207 @@
...
+        <p>We&rsquo;re reinventing how we scale by becoming simpler and modular, similar to how applications have evolved in cloud data centers. Open source and OpenStack innovations represent a unique opportunity to meet these requirements and Canonical&rsquo;s cloud and open source expertise make them a good choice for AT&T.</p>
+        <cite class="p-pull-quote__citation">Toby Ford, Assistant Vice President of Cloud Technology, Strategy and Planning at AT&T</cite>
+      </blockquote>
+    </div>
+  </div>
+</div>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-12">
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: Maybe make the logo more narrow so there is not one on its own.

File: templates/telecommunications/index.html (hunk 11 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done

File: templates/telecommunications/index.html (hunk 12 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done

File: templates/telecommunications/index.html (hunk 13 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done

File: templates/telecommunications/index.html (hunk 10 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Added, requested link in copy doc

**Comment by grahambancroft** [comment]
@anthonydillon all changes made, this is ready for review again

File: templates/telecommunications/index.html [hunk 14]
```diff
@@ -0,0 +1,208 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>A telco-grade managed cloud</h2>
+      <p>A number of telcos rely on Canonical&rsquo;s fully managed cloud service. Get Telco Grade SLAs and predictable operational costs for your OpenStack cloud, on your servers at your locations. We build, operate, scale and upgrade your cloud, while you concentrate on building your applications in flexible and agile environments.</p>
+      <p><a href="/cloud/managed-cloud">Find out more<span class="u-off-screen"> about Canonical&rsquo;s Foundation Cloud Build</span>&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: Isn't this link to Canonical's managed cloud. Not Foundation Cloud Build?

File: templates/telecommunications/index.html [hunk 15]
```diff
@@ -0,0 +1,208 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>NFVI cloud build</h2>
+      <p>Canonical&rsquo;s Foundation Cloud Build for Telcos is a pre-tested, pre-integrated OpenStack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about NFVI cloud build</span>&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/telecommunications/index.html]: I know this matches the copy doc but there is no `NFVI` on the foundation-cloud. I'll look into it.

File: templates/telecommunications/index.html (hunk 14 above)
Comment (grahambancroft) [reply templates/telecommunications/index.html]: Done

File: navigation.yaml [hunk 16]
```diff
@@ -23,6 +23,8 @@ cloud:
       path: /cloud/training
     - title: Financial services
       path: /financial-services
+    - title: telcos
```
Comment (anthonydillon) [inline navigation.yaml]: This should be `Telcos` please

**Comment by frankiegranato** [comment]
- [ ] Add final CTA row at foot of page
- [ ] Add contact us link (https://www.ubuntu.com/cloud/contact-us)

**Pull request merged**